owlbot.py

```diff
@@ -31,10 +31,11 @@
 extras_by_python = {
     "3.8": ["tests", "alembic"],
     "3.9": ["tests", "geography"],
+    "3.10": ["tests", "geography"],
 }
 templated_files = common.py_library(
-    unit_test_python_versions=["3.6", "3.7", "3.8", "3.9"],
-    system_test_python_versions=["3.8", "3.9"],
+    unit_test_python_versions=["3.6", "3.7", "3.8", "3.9", "3.10"],
+    system_test_python_versions=["3.8", "3.9", "3.10"],
```

Comment:
As above (`noxfile.py` is actually generated from `owlbot.py`).

Reply:
Changed to only run on 3.8 and 3.10